Kudu.Core/Functions/KedaFunctionTriggerProvider.cs

```diff
@@ -216,7 +216,7 @@ internal static IEnumerable<ScaleTrigger> GetStandardScaleTriggers(IEnumerable<F
                     IKedaAuthRefProvider authProvider = getTriggerAuthProvider(triggerType);
                     if (authProvider != null)
                     {
-                        scaleTrigger.AuthenticationRef = authProvider.PopulateAuthenticationRef(function.Binding, function.FunctionName);
+                        scaleTrigger.AuthenticationRef = authProvider.PopulateAuthenticationRef(function.Binding, function.FunctionName, functionNamespace);
```

Comment:
nitpick: I think it would be a good idea to put the functionName and Namespace in an object and pass around for the entire flow. This way we don't need to change multiple method signatures further when we need to add/modify more properties. Eg on top of my mind for such property is AppType etc etc. 

P.S. Doesn't have to go in this PR.

Reply:
good suggestion, will take it in another PR